testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestReadUniontype.java

```diff
@@ -49,27 +51,13 @@ public static Object[][] storageFormats()
         return new String[][] {{"ORC"}, {"AVRO"}};
     }
 
-    private void createTestTable(String storageFormat)
-    {
-        cleanup();
-        onHive().executeQuery(format(
-                "CREATE TABLE %s (id INT,foo UNIONTYPE<" +
-                        "INT," +
-                        "DOUBLE," +
-                        "ARRAY<STRING>>)" +
-                        "STORED AS %s",
-                TABLE_NAME,
-                storageFormat));
-    }
-
     @Test(dataProvider = "storage_formats", groups = SMOKE)
     public void testReadUniontype(String storageFormat)
     {
         // According to testing results, the Hive INSERT queries here only work in Hive 1.2
         if (getHiveVersionMajor() != 1 || getHiveVersionMinor() != 2) {
             throw new SkipException("This test can only be run with Hive 1.2 (default config)");
         }
-        createTestTable(storageFormat);
```

Comment:
Won't the insert below fail without the table existing?

Reply:
Right, fixed.